task-launcher/src/tasks/math/trials/sliderStimulus.js

```diff
@@ -3,12 +3,32 @@ import _shuffle from 'lodash/shuffle';
 import _toNumber from 'lodash/toNumber';
 import { jsPsych, isTouchScreen } from '../../taskSetup';
 import { camelize } from '@bdelab/roar-utils';
-import { arrowKeyEmojis, isPractice, setSkipCurrentBlock, taskStore } from '../../shared/helpers';
+import { arrowKeyEmojis, isPractice, setSkipCurrentBlock, taskStore, replayButtonSvg, setupReplayAudio, PageAudioHandler } from '../../shared/helpers';
+import { mediaAssets } from '../../..';
 
 let chosenAnswer,
   sliderStart,
   keyboardResponseMap = {};
 
+  function setUpAudio(contentWrapper, prompt, responseType) {
+    // add replay button
+    const replayButton = document.createElement('button'); 
+    replayButton.innerHTML = replayButtonSvg;
+    replayButton.id = 'replay-btn-revisited';
+    replayButton.classList.add('replay'); 
+    replayButton.disabled = true;
+    contentWrapper.insertBefore(replayButton, prompt);
+  
+    const cue = responseType === 'button' ? 'numberLinePrompt1' : 'numberLineSliderPrompt1';
+  
+    const audioFile = mediaAssets.audio[cue] || '';
+    // Returns a promise of the AudioBuffer of the preloaded file path.
+    PageAudioHandler.playAudio(audioFile, () => {
+      // set up replay button audio after the first audio has played
+      setupReplayAudio(cue);
```

Comment:
Maybe check for `cue` to have length and only then call setupReplayAudio. So something like:
```
if (cue) {
   setupReplayAudio(cue);
}
```

Reply:
I'll make that change, thanks!